### ISSUE TITLE
Preview Size

### ISSUE THREAD

**Issue by arielaco** [jwortmann/quick-view#4]
Is it possible to add this? Where could I modify it on the code if I'd like to contribute?

I'm interested in choosing to see the preview in either full size, a % or a fixed max size.

**Comment by jwortmann** [comment]
The popup size for images in hardcoded in https://github.com/jwortmann/quick-view/blob/3d4415de7545d8d234a7735bc6c3fe97eaec3c76/quick_view.py#L22-L23

i.e. it currently uses 200 pixel (multiplied by a scaling factor dependent on the font size) for the image width (if width > height), or less than 200, but at least 100 pixel for the width (if height > width). (Assuming I have no bug somewhere.)

Would you prefer to always have a larger popup size, or only for example after clicking a button that could be added to the popup? I was considering to add a small button next to the "width x height pixels" label, that you can click to open an image at full size in a new tab, but I have not implemented it yet. Feedback would be helpful for this.

**Comment by jwortmann** [comment]
I've added a small icon for image previews which can be clicked to open an image in a new tab. This is enabled by default, but can be disabled in the settings by removing the *"open_image_button"* value from the *"popup_style"* option.

For image formats that require a conversion (e.g. SVG), it only works on ST4.

I've just made a new release, but it will take a few hours until Package Control will notice it.
